components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/manager/FxaAccountManager.kt

```diff
@@ -150,7 +150,7 @@ open class FxaAccountManager(
     private val logger = Logger("FirefoxAccountStateMachine")
 
     @Volatile
-    private var latestAuthState: String? = null
+    var latestAuthState: String? = null
```

Comment:
Consider adding `@VisibleForTesting` annotation.

Reply:
But actually, after reading the test, I don't think you need to make this public at all. Existing tests which ensure we validate auth state don't need  this to be public.

Reply:
I did figure out how to set that value without making that variable public via some spelunking through the code. That has been fixed.